spec/index.bs

```diff
@@ -764,9 +764,9 @@ The <dfn dfn>physical size</dfn> of a [=texture subresource=] is the dimension o
 [=texture subresource=] in texels that includes the possible extra paddings
 to form complete [=texel blocks=] in the [=subresource=].
 
-  - For pixel-based {{GPUTextureFormat}}s, the [=physical size=] is always equal to the size of the [=texture subresource=]
+  - For pixel-based {{GPUTextureFormat|GPUTextureFormats}}, the [=physical size=] is always equal to the size of the [=texture subresource=]
     used in the sampling hardwares.
-  - {{GPUTexture}}s in block-based compressed {{GPUTextureFormat}}s always have a [=mipmap level=] 0 whose {{GPUTexture/[[textureSize]]}}
+  - {{GPUTexture|GPUTextures}} in block-based compressed {{GPUTextureFormat|GPUTextureFormats}} always have a [=mipmap level=] 0 whose {{GPUTexture/[[textureSize]]}}
```

Comment:
```suggestion
  - [=Textures=] in block-based compressed {{GPUTextureFormat|GPUTextureFormats}} always have a [=mipmap level=] 0 whose {{GPUTexture/[[textureSize]]}}
```

We have this definition for the internal texture concept (as opposed to the JS interface), at least for now, so can use it.